guides/quickstart.md

```diff
@@ -0,0 +1,175 @@
+# Quickstart: How to configure and run the analyzer
+
+The code for this example is available in the `go-flow-levee/guides/quickstart` directory.
+
+## Context
+
+Suppose you have the following piece of code:
+
+`auth.go`
+```go
+package auth
+
+import "log"
+
+type Authentication struct {
+	Username string
+	Password string
+}
+
+func authenticate(auth Authentication) (*AuthenticationResponse, error) {
+	response, err := makeAuthenticatedRequest(auth)
+	if err != nil {
+		log.Printf("unable to make authenticated request: incorrect authentication? %v\n", auth)
+		return nil, err
+	}
+	return response, nil
+}
+```
+
+This code is an instance of [CWE-522](https://cwe.mitre.org/data/definitions/522.html), "Insufficiently Protected Credentials".
+The `Password` field on the `Authentication` struct contains credentials, which should not be allowed to reach logs.
+
+This example may seem contrived, but the analyzer has actually caught many similar cases in the wild.
+
+## Configuration
+
+Let's see how we can configure the analyzer to automatically detect this incorrect handling of credentials.
+
+In order to do its job, the analyzer needs 2 pieces of information:
+1. Which types hold sensitive data? (Sources)
+2. Which functions should not be called with sensitive data? (Sinks)
+
+Given this information, the analyzer will perform taint propagation to determine whether sensitive values can reach calls to functions that they shouldn't reach.
+
+### Configuring a Source type
+
+In taint propagation lingo, a value that contains data of interest is called a Source.
+In the current example, a Source is a value that contains sensitive data, e.g. a password.  
+
+There are 2 ways to define a Source type:
+1. Define a field tag and apply it to sensitive fields
+2. Describe a type using its name, the name of its package, and the relevant field(s)
+
+#### Using field tags
+
+In the analyzer's configuration, define the field tag:
+
+`analyzer_configuration.yaml`
+```yaml
+FieldTags:
+  - Key: datapolicy
+    Val: secret
+```
+
+In the code you want to analyze, add the tag to the sensitive field:
+
+`auth.go`
+```go
+type Authentication struct {
+  Username string 
+  Password string `datapolicy:"secret"`
+}
+```
+
+If you do not wish to define your own field tag, you may use the built-in `levee:"source"` tag.
+
+#### Using type descriptions
+
+In the analyzer's configuration, identify the sensitive data:
+
+`analyzer_configuration.yaml`
+```yaml
+Sources:
+  - Package: auth
```

Comment:
This feels a little misleading, because `Package` expects a complete path to the package, not just the tail of it.  Using this in the example config instead of the `FieldTags` does not produce a diagnostic.

I know you link the `configuration/README.md` right after, but consider expanding this to `Package: complete/path/to/auth`.  I would prefer to leave the partial / regexp matching to be explained in the `configuration/README.md` and not complicate that here.

This is a weird one, since we give the `package auth` in the example above, but without context of surrounding `GOPATH` / `go.mod`, the package path is incomplete.  I suppose you could use the complete path `github.com/google/go-flow-levee-guide/guides/quickstart/auth`, since the code is being introduce here.  If you go that route, maybe add the path as a comment to the `package auth`, i.e. 
```golang
// complete path github.com/ ...
package auth
```